Lottie-Windows/Lottie-Windows.csproj

```
@@ -1,6 +1,14 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
 
   <PropertyGroup>
+    
+    <!-- Description -->
+    <Title>Lottie-Windows</Title>
+    <Description>
+      Render Adobe AfterEffects animations natively on Windows: aka.ms/lottie
```

I'd put the full link so that it's more likely to be clickable depending on client.

```suggestion
      Render Adobe AfterEffects animations natively on Windows: https://aka.ms/lottie
```